pkgs/applications/misc/megasync/default.nix

```
@@ -81,46 +116,24 @@ mkDerivation rec {
   dontUseQmakeConfigure = true;
   enableParallelBuilding = true;
 
-  preConfigure = ''
-    cd src/MEGASync/mega
-    ./autogen.sh
-  '';
-
-  configureFlags = [
-    "--disable-examples"
-    "--disable-java"
-    "--disable-php"
-    "--enable-chat"
-    "--with-cares"
-    "--with-cryptopp"
-    "--with-curl"
-    "--with-ffmpeg"
-    "--without-freeimage"
-    "--without-readline"
-    "--without-termcap"
-    "--with-sodium"
-    "--with-sqlite"
-    "--with-zlib"
+  cmakeFlags = [
+    (lib.cmakeBool "USE_PDFIUM" false) # PDFIUM is not in nixpkgs
+    (lib.cmakeBool "USE_FREEIMAGE" false) # freeimage is insecure
+    (lib.cmakeBool "ENABLE_DESIGN_TOKENS_IMPORTER" false) # cannot be fetched
   ];
 
-  postConfigure = ''
-    cd ../..
-  '';
-
-  preBuild = ''
-    qmake CONFIG+="nofreeimage release" MEGA.pro
-    pushd MEGASync
-      lrelease MEGASync.pro
-      DESKTOP_DESTDIR="$out" qmake PREFIX="$out" -o Makefile MEGASync.pro CONFIG+="nofreeimage release"
-    popd
+  postInstall = ''
+    wrapProgram $out/bin/megasync --prefix PATH : ${lib.makeBinPath [ xorg.xrdb ]}
```

```suggestion
  preFixup = ''
    qtWrapperArgs+=(--prefix PATH : ${lib.makeBinPath [ xorg.xrdb ]})
```
I had another look and apparently the Qt wrapper already wraps the program. To avoid double wrapping we should only add our custom arguments to the Qt wrapper.